server/text_generation_server/models/causal_lm.py

```diff
@@ -92,7 +92,6 @@ def biggest_single_chunk(offset):
         return 0
 
 
-@torch_compile_for_eager
```

Comment:
why don't use torch._dynamo.graph_break() in place of mark_step instead of removing compilation of graph?
same question applies to all below cases

Reply:
does the change cause the ops within the function to be executed eagerly?

Reply:
> does the change cause the ops within the function to be executed eagerly?
I assume(I have not run logging of fallback to eager events) that functions excluded from torch compile regions (as done in this PR) are now running eager e.g. pytorch ops from code that got torch compile decorator discarded are running eager .

Reply:
In some of internal testing it was revealed that excluding those functions(as in this PR) from torch compile region did not have an impact on performance or accuracy.